src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
I still don't get what you're saying sorry. It seems fairly clear that If
undefined means X, on the other side X can be translated to undefined,
cause it will translate back to X.

Are you talking a simple redemption case with no Paths? It ought to still
work?